BrownField/Networking/Concepts/Connectivity-Multi-region-with-vWAN/Dual-Region-with-vWAN.md

```diff
@@ -0,0 +1,122 @@
+
+## Dual-region with Secure vWAN scenario  
+Secure vWAN with Routing Intent provides the capability to send all Internet traffic and Private network traffic (RFC 1918) to a security solution like Azure Firewall, a third-party NVA, or SaaS. 
+In the scenario, we have a network topology that spans two regions. There is one vWAN with two Hubs, Hub1 and Hub2. Hub1 is in Region 1, and Hub2 is in Region 2.Each Hub in both regions has its instance of Azure Firewall deployed(Hub1Fw, Hub2Fw), essentially making them Secure vWAN Hubs. Having Secure vWAN hubs is a technical prerequisite to Routing Intent. Secure vWAN Hub1 and Hub2 have Routing Intent enabled.  
+
+Each region has its own AVS Private Cloud and an Azure Vnet. There is also an on-premise site connecting to both regions, which we will review in more detail later in this document.
+![image](https://github.com/jasonamedina/Enterprise-Scale-for-AVS/assets/97964083/0d2d0b80-e550-4f69-a321-411658a066ab)
+### Understanding Topology Connectivity 
+**Brown Connections**: AVS private cloud connection to its local regional hub.  
+**Pink Connections**: AVS private cloud connection to its cross-regional hub.   
+**Orange Connection**: AVS Region 1 Global Reach connection back to on-premise.   
+**Green Connection**: AVS Region 2 Global Reach connection back to on-premise.   
+**Purple Connection**: AVS Region 1 and AVS Region 2 Global Reach connection back to each other's private cloud.   
+**Black Connections**: On-premise connectivity via ExpressRoute to both regional hubs.  
+**Inter-Hub Connection**: When two hubs are deployed under the same vWAN, they automatically form an inter-hub connection with one another. The purpose of the inter-hub is to transit cross-regional traffic between the two hubs.  
+
+## Dual-region Secure vWAN Traffic Flows
+
+The following sections below will discuss traffic flows and connectivity for AVS, On-Premise, Azure Vnets, and the Internet. 
+
+### Azure VMware Solution cross-region connectivity & traffic flows
+
+This section will focus on only the AVS Cloud Region 1 and AVS Cloud Region 2. Each AVS private cloud will have an ExpressRoute connection to its local regional hub (brown connections) and an ExpressRoute connection to the cross-regional hub (pink connections).
+
+Each AVS Cloud Region connects back to on-premise via Global Reach. AVS Cloud Region 1 Global Reach connection is shown in orange as "Global Reach (A)". AVS Cloud Region 2 Global Reach connection is shown in green as "Global Reach (B)". Both AVS private clouds are connected directly to each other via Global Reach shown in purple as Global Reach (C). Keep in mind that Global Reach traffic will never transit any hub firewalls. See traffic flow section below for more information.  
+
+The diagram also depicts how all routes in each AVS Cloud region will learn routes from both the local and cross-regional hub. All blue routes are from Region 1, and all red routes are from Region 2. 
```

Comment:
Fix this sentence that does not read OK: "how all routes in each AVS Cloud region will learn routes from both the local and cross-regional hub"

Reply:
@fguerri, I corrected the verbiage as shown highlighted in the screenshot.

![image](https://github.com/Azure/Enterprise-Scale-for-AVS/assets/97964083/df523573-9c32-4b1b-b15b-08dd6766c177)